build-system/tasks/performance/measure-documents.js

```diff
@@ -118,6 +179,80 @@ const readMetrics = (page) =>
     };
   });
 
+/**
+ * Set up appropriate handlers and default handlers
+ * @param {?Object} handlerOptions
+ * @param {string} handlerName
+ * @param {Puppeteer.page} page
+ */
+function setUpRequestHandler(handlerOptions, handlerName, page) {
+  // Default handlers
+  const handlers = [analyticsVendorRewriteHandler];
```

Comment:
Yup, it's default because we should want to keep control and experiment as similar as possible (configs shouldn't change between versions). We can setup the default handler inside measure documents and add an explanation.